database/postgresql/src/main/java/com/example/javaopencommerce/catalog/ItemQueryRepositoryImpl.java

```diff
@@ -3,8 +3,10 @@
 import com.example.javaopencommerce.catalog.dtos.FullItemDto;
 import com.example.javaopencommerce.catalog.dtos.ItemDto;
 
+import javax.enterprise.context.ApplicationScoped;
 import java.util.List;
 
+@ApplicationScoped
```

Comment:
Not a fan of that annotation. It would be better to create bean with a @Producer.